GoodThingsComein3s/ViewControllers/SignUpLoginViewController.m

```diff
@@ -6,11 +6,47 @@
 //
 
 #import "SignUpLoginViewController.h"
+#import "LoginViewController.h"
+#import "SignUpViewController.h"
 
-@interface SignUpLoginViewController ()
+
+@interface SignUpLoginViewController ()<LoginViewControllerDelegate,SignUpViewControllerDelegate>
 
 @end
 
 @implementation SignUpLoginViewController
 
+- (IBAction)signUpViewControllerDidTapLogin:(id)sender {
+    [self performSegueWithIdentifier:@"loginSegue" sender:nil];
+}
+
+
+- (IBAction)signUpViewControllerDidTapSignUp:(id)sender {
+    [self performSegueWithIdentifier:@"signUpSegue" sender:nil];
+}
+
+- (void)dismissLoginSignUpFromLogin {
+    [self dismissLoginSignUp];
+}
+
+- (void)dismissLoginSignUpFromSignUp {
+    [self dismissLoginSignUp];
+}
+
+- (void)dismissLoginSignUp {
+    [self dismissViewControllerAnimated:YES completion:^{
+        NSLog(@"signUpLoginVC dismissed");
+    }];
+}
```

Comment:
Since we want to remove NSLog calls before committing, this wrapper method could probably be removed if it's not doing anything else. And directly call 
`[self dismissViewControllerAnimated:YES completion:nil]` instead